pandas/core/generic.py

```diff
@@ -4260,16 +4271,76 @@ def _get_values(self):
         return self.values
 
     def get_values(self):
-        """same as values (but handles sparseness conversions)"""
+        """
+        Same as values (but handles sparseness conversions).
```

Comment:
Let's avoid reference to .values in the first line.

    Return a NumPy representation of the data after converting sparse to dense.

And then we can use the `See Also` to mention `.values`